rtc/getting-started.md

```diff
@@ -6,44 +6,86 @@ description: Getting started with RTC
 keywords: rtc
 ---
 
-{% assign beta_feature = "The Real-Time Collaboration (RTC) plugin" %}
-{% assign pre-release_type = "Open Beta" %}
-{% include misc/beta-note.md %}
+{% assign pluginname = "Real-Time Collaboration (RTC)" %}
+{% assign plugincode = "rtc" %}
 
-## Prerequisites
+{{site.requires_5_9v}}
 
-* A {{site.cloudname}} API key from [{{site.accountpage}}]({{site.accountsignup}}).
-* A JSON Web Token (JWT) key generated at [{{site.accountpageurl}}]({{site.accountpageurl}}). Temporarily store the generated keys for use in the following procedure.
-* {% include misc/secure-context.md %}
+This procedure will assist with setting up {{site.productname}} with real-time collaboration.
 
-## Basic Real-Time Collaboration (RTC) setup
+> **Important**: {% include misc/secure-context.md %}
 
-The following example assumes you are familiar with integrating TinyMCE; if you are new TinyMCE user please follow the [basic setup guide]({{site.baseurl}}/general-configuration-guide/basic-setup/) first.
+The steps required for setting up Real-Time Collaboration for {{site.productname}} are:
 
-This example shows how to get the Real-Time Collaboration (RTC) plugin up and running using hardcoded values. For a production setup, the document ID and secret would be retrieved from your server. This example also assumes that a JWT provider endpoint exists at '/jwt'. For information on setting up a JWT endpoint, see: [JWT authentication]({{site.baseurl}}/rtc/jwt-authentication/).
+1. Register for a {{site.cloudname}} API key (paid plan required).
+1. Add a public key to the {{site.cloudname}} API key.
+1. Set up an Encryption Provider endpoint (or server).
+1. Set up a JSON Web Token (JWT) Provider endpoint (or server).
+1. Configure the required {{site.productname}} options.
+
+## Register for a {{site.cloudname}} API key
```

Comment:
I wonder if prepending the step number on each heading makes it more clear that these are the specific steps an integrator should go through.

Reply:
yes it would, but if the steps change, links break. I'll circle back if I have time.

Reply:
Added in https://github.com/tinymce/tinymce-docs/pull/2091/commits/96dcc9060c18a88185cde11be51711a2defaeb43